src/Symfony/Component/ClassLoader/ClassMapGenerator.php

```diff
@@ -134,7 +134,7 @@ private static function findClasses($path)
                     }
 
                     if ($isClassConstant) {
-                        continue;
+                        break;
```

Comment:
Yes, continue inside switch behaves differently in PHP than in other languages.

http://php.net/manual/en/control-structures.continue.php
```
Note: In PHP the switch statement is considered a looping structure for the purposes of continue. continue behaves like break (when no arguments are passed). If a switch is inside a loop, continue 2 will continue with the next iteration of the outer loop.
```